hiku/expr/checker.py

```diff
@@ -132,7 +132,7 @@ def visit_tuple_generic(self, node):
         assert isinstance(sym, Symbol), type(sym)
         args = [self.visit(val) for val in node.values[1:]]
         fn_type = node_type(self.types, sym)
-        assert len(fn_type.__arg_types__) == len(args), 'Wrong arguments count'
+        assert len(fn_type.__arg_types__) == len(args), f'Wrong arguments count at "{sym.name}" node'
```

Comment:
Seems like `sym.name` is not a node's name, it is a function's name. Could you rephrase this error?

Reply:
And minimum supported Python version is 3.5, so we can't use f-strings yet